www/src/css/headline.css

```diff
@@ -8,7 +8,7 @@
   margin:0 0 0 10px
 }
 
-.b-headline .b-headline__store-link {
+.region-installed .b-headline__store-link {
```

Comment:
OK. We discussed the nesting issue in couple PRs already, so I understand that you addressed it wherever was possible and reasonable. So if that needs to stay I'm OK with that.